clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -513,7 +513,52 @@ LogicalResult CastOp::verify() {
   llvm_unreachable("Unknown CastOp kind?");
 }
 
+Value fold_casts(llvm::SmallVector<CastOp>& ops) {
```

Comment:
These functions should follow the code style: https://llvm.github.io/clangir/GettingStarted/coding-guideline.html, e.g. `foldCasts`.